docs/scos/dev/upgradability-services/upgradability-guidelines.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Upgradability guidelines
+description: Reference information for evaluator and upgrader tools.
+last_updated: Nov 25, 2021
+template: concept-topic-template
+---
+
+
+## A core method is used on the project level
+
+Modules have public and private APIs. The public API includes the entities like facade, plugin stack, dependency list, etc. This check covers private API entities, including Business model, Factory, Dependency provider, Repository, and Entity manager.
+
+While public API updates always support backward compatibility, private API updates can break backward compatibility. So, backward compatibility in minor releases is not guaranteed in the private API. For example, if you use a core method on the project level, and it is updated or removed, it can cause unexpected issues durning minor updates.
```

Comment:
```suggestion
While public API updates always support backward compatibility, private API updates can break backward compatibility. So, backward compatibility in minor releases is not guaranteed in the private API. For example, if you use a core method on the project level, and it is updated or removed, it can cause unexpected issues during minor updates.
```